yarn_api_client/main.py

```diff
@@ -16,6 +17,10 @@ def get_parser():
         description='Client for Hadoop® YARN API')
 
     parser.add_argument('--endpoint', help='API endpoint (https://test.cluster.com:8090)')
+    #parser.add_argument('--api_class', help='Please provide api class - rm, hs, nm, am', required=True)
```

Comment:
Is this intentionally commented out?  If so, perhaps an additional comment as to why or when it should be uncommented would be helpful.